.github/ISSUE_TEMPLATE/pre-release-demos-nightly.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Pre-Release Demo Testing on Nightly
+about: This template can be used to track the testing of demos on nightly leading up to the next Stackable release
+title: "chore(tracking): Test demos on nightly versions"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    Make sure to update the link in '.github/ISSUE_TEMPLATE/release.md' when
+    you change the front matter above.
+-->
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
+-->
+
+## Pre-Release Demo Testing on Nightly
+
+Part of <https://github.com/stackabletech/issues/issues/TRACKING_ISSUE>
+
+For each demo, run the following commands:
+
+```shell
+# Install demo
+stackablectl demo <DEMO_NAME>
+
+# Uninstall operators
+stackablectl release uninstall <CURRENT_RELEASE>
```

Comment:
```suggestion
# Get a list of operators that are installed
kubectl -n stackable-operators get deploy -o json | jq -r '.items[].metadata.name' | sed -e 's/-operator-deployment//' | sort

# Uninstall operators
stackablectl release uninstall <CURRENT_RELEASE>
```

Reply:
Or, just use `stackablectl operator installed` :)

Reply:
Done in 3a9ef57

Reply:
True, although for me there is too much noise.

Perhaps `stackablectl operator installed --output=plain` is a nice compromise?

Reply:
I have added the suggestion in another review. Will resolve this again.